boards/adafruit-clue/Makefile.include

```diff
@@ -0,0 +1,34 @@
+# Bossa is the default programmer
+PROGRAMMER ?= nrfutil
+
+ifeq ($(PROGRAMMER),nrfutil)
+  # The preinstalled Arduino bootloader must also be taken into account so
+  # ROM_OFFSET skips the space taken by such bootloader.
+  ROM_OFFSET = 0x26000
+
+  NRFUTIL_ARDUINO_PREFLASH = yes
+  PREFLASH_DELAY = 1
+
+  ifneq (,$(filter reset flash flash-only, $(MAKECMDGOALS)))
+    # Add 2 seconds delay before opening terminal: this is required when opening
+    # the terminal right after flashing. In this case, the stdio over USB needs
+    # some time after reset before being ready.
+    TERM_DELAY = 2
+    TERMDEPS += term-delay
+  endif
+
+  include $(RIOTMAKE)/tools/nrfutil.inc.mk
+endif
+
+term-delay:
+	sleep $(TERM_DELAY)
+
+TESTRUNNER_CONNECT_DELAY ?= 2
+$(call target-export-variables,test,TESTRUNNER_CONNECT_DELAY)
+
+USB_VID ?= 1209
+USB_PID ?= 7D00
+
+CFLAGS += -DCONFIG_USB_VID=0x$(USB_VID) -DCONFIG_USB_PID=0x$(USB_PID)
```

Comment:
This isn't necessary anymore thanks to https://github.com/RIOT-OS/RIOT/pull/13148

Reply:
Indeed, fixed.